src/watchmaker/static/config.yaml

```diff
@@ -1,11 +1,12 @@
+watchmaker_version: "== 0.16.6"
```

Comment:
How are we maintaining this pin? I think this can at least be:
```suggestion
watchmaker_version: ">= 0.17"
```

Reply:
I used `~= 0.17.0` which will limit to 0.17.x.